src/tmpfiles/tmpfiles.c

```diff
@@ -2633,6 +2740,42 @@ static int find_gid(const char *group, gid_t *ret_gid, Hashmap **cache) {
         return name_to_gid_offline(arg_root, group, ret_gid, cache);
 }
 
+static int parse_age_by_from_arg(const char *age_by_str, AgeBy *ret_age_by) {
+        _cleanup_free_ char *a = NULL;
+        char *s = NULL;
+        AgeBy ab = AGE_BY_UNSET;
+
+        assert(age_by_str);
+        assert(ret_age_by);
+
+        a = strdup(age_by_str);
+        if (!a)
+                return -ENOMEM;
```

Comment:
why copy this? sounds unecessary?

Reply:
Oh, this is because `strstrip()` modifies the string...

Reply:
ignoring whitespace while parsing sounds trivial